counterpartylib/lib/micropayments/__init__.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
```

Comment:
Removed the headers as requested. I am also no lawer, but as far as I can see we are both using the same MIT license. So we should have no issues there.